openlibrary/plugins/admin/code.py

```diff
@@ -387,7 +388,7 @@ def get_stats(self, today):
 class ipstats:
     def GET(self):
         web.header('Content-Type', 'application/json')
-        json = urllib.request.urlopen("http://www.archive.org/download/stats/numUniqueIPsOL.json").read()
+        json = requests.get("http://www.archive.org/download/stats/numUniqueIPsOL.json").content
         return delegate.RawText(json)
```

Comment:
```suggestion
        json_bytes = requests.get("http://www.archive.org/download/stats/numUniqueIPsOL.json").content
        return delegate.RawText(json_bytes)
```

Reply:
@cclauss I am actually not sure how to check what `delegate.RawTest()` is expecting. Can you please let me know how to go on about checking it?

Reply:
I would say that we want `.text`

% `grep -e RawText -e rawtext **/*.py`
```
infogami/core/code.py:        return delegate.RawText(render.feed(site, changes))
infogami/plugins/api/code.py:        return delegate.RawText(out, content_type=content_type)
infogami/utils/app.py:        from infogami.utils.delegate import RawText
infogami/utils/app.py:        return RawText(simplejson.dumps(data))
infogami/utils/delegate.py:        out = RawText("")
infogami/utils/delegate.py:    if hasattr(out, 'rawtext'):
infogami/utils/delegate.py:        html = out.rawtext
infogami/utils/delegate.py:class RawText(web.storage):
infogami/utils/delegate.py:        web.storage.__init__(self, rawtext=text, **kw)
```

Reply:
Yes. 
```
infogami/utils/delegate.py:        out = RawText("")
```
This makes it very clear. Thanks @cclauss 
Made the changes.